src/index.js

```diff
@@ -554,7 +554,17 @@ class EliteDangerousJournalServer {
     if (event === 'change' && filepath === this.currentJournal) {
       this.getJournalUpdate();
     } else {
-      console.log(`${chalk.green(`Filesystem event ${chalk.red(event)} occured for`)} ${chalk.magenta(filepath)}`);
+      console.log(`${chalk.green(`Filesystem event ${chalk.red(event)} occurred for`)} ${chalk.magenta(filepath)}`);
+      const baseName = path.basename(filepath).toLowerCase();
+      if (baseName === 'status.json') {
+        const statusLines = fs.readFileSync(filepath, 'utf-8').split('\n').filter(item => item.trim());
+        if (statusLines && statusLines.length) {
+          const formattedStatus = JSON.parse(statusLines[0]);
+          if (formattedStatus) {
+            this.broadcast(formattedStatus);
```

Comment:
Does the status have something to denote that it's coming from the `status.json` file and not the standard Journal?

If not, should we add some property to our broadcasts like `channel` that defines whether it is coming from the Journal or the `status.json` file?

Reply:
See above, using `{ "event": "X-Status", ... }`?